NUMBER_NEURON: 6
VOLTAGE_TRESHOLD: 1.2
DELTA: 0.0001
LEARNING_RATE: 0.1
LEAK_RATE: 1.5
TIME_ITERATION: 400.0
ITERATION_SPIKE_IN: 5

VOLTAGE_TRESHOLD_TEST: [1.1,1.1,1.1,1.1,1.1, 1.1, 1.1]
LEAK_RATE_TEST: [2.0,1.5,1.8,2.0,2.5, 3.0, 3.5]


NAME_FILE_REWARD: "/statistics/reward_change_new_leak_rate_new_odom7.txt"
NAME_FILE_LEGS: "/statistics/legs_change_new_leak_rate_test_new_odom7.txt"
NAME_FILE_VOLT: "/statistics/voltage_change_new_leak_rate_test_new_odom7.txt"



